README.md

```diff
@@ -65,12 +65,20 @@ To build the project:
 - Add dependency on Scala compiler plugin [Macro Paradise](https://github.com/scalamacros/paradise).
   - This can be done by adding `addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)` to the build settings in build.sbt.
   - An example project, which uses Macro Paradise can be found [here](https://github.com/scalamacros/sbt-example-paradise).
-- Add dependency on this repository.
-  - This can be done by adding the following line to the build.sbt file
-  ```scala
-  lazy val hiddenArgs = ProjectRef(uri("https://github.com/keddelzz/hidden-args.git"), "hiddenargs")
+- Add the following resolver to your `build.sbt`:
+```
+resolvers += Resolver.bintrayRepo("keddelzz", "maven")
```

Comment:
I recommend it as a replacement. Although the personal repo is perfectly okay for some first publications, JCenter is a more mature solution. It's a second (after Maven Central) well-known standard JVM repository.